knowledge/studios/bbc_studios/qna.yaml

```diff
@@ -0,0 +1,46 @@
+created_by: 
+domain: television
+seed_examples:
+- answer: |
+    The BBC Studios production division was formed in 2016 and launched as a commercial entity in 2017,[6] enabling it to produce programming for other broadcasters and services to generate profit to return to the BBC to supplement licence fee income. In exchange, the BBC agreed to place production of much of its non-news programmes to tender, allowing third-party independents to compete with BBC Studios on bids to produce them.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!